awesomemotive/affiliatewp-order-details-for-affiliates
# Issue/81 - Bump WordPress minimum requirements

Issue #81


**Housekeeping**


**Proposed changes**


## How to test it
1. Install and activate Order Details for Affiliates add-on
2. Confirm the plugin works as expected without any PHP errors.
3. Confirm existence and test saving the new settings on `Affiliates > Settings > Integrations`
